INSTALL.rst

```diff
@@ -30,14 +29,24 @@ GROMACS_.
 Conda environment with pre-requisites
 -------------------------------------
 
-We make a conda environment with the latest packages for Python 2.7
+To make a conda environment with the latest packages for Python 2.7
 and name it *mdpow*; this installs the larger dependencies that are
 pre-requisites for MDPOW::
 
  conda create -c conda-forge -n mdpow python=2.7 numpy scipy 'matplotlib<3.3' 'mdanalysis<2' 'mdanalysistests<2' pyyaml
  conda activate mdpow  
  pip install gromacswrapper
 
+
+For Python 3.7 and up.
+
+*Note:* with Pandas version 1.3 there is an error with `Alchemlyb <https://github.com/alchemistry/alchemlyb>`_
+(see `issue #147 <https://github.com/alchemistry/alchemlyb/issues/147>`_) which will be fixed in Alchemlyb 0.5.::
```

Comment:
note: technically, all that alchemlyb 0.5.0 does is to not allow 1.3.0 as a dependency

We can leave your Note as is, though.